pkg/operator/tracker.go

```diff
@@ -43,15 +43,19 @@ func (t *tprTracker) add(name, version string) {
 func (t *tprTracker) remove(name string) {
 	delete(t.clusterRVs, name)
 
+	t.Lock()
```

Comment:
no, the only conflict here would be with the `stop` function where the channel is closed and removed from that map